src/components/i18n/Plural.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from 'react';
+
+type Props = {
+    /** The plural category this string. (required) */
+    category: 'zero' | 'one' | 'two' | 'few' | 'many' | 'other',
```

Comment:
Seems like this should still use the `CATEGORY_ONE` constant types to avoid duplicating them here.

Reply:
Flow does not allow that.